### ISSUE TITLE
Cloud authentication is not functional

### ISSUE THREAD

**Issue by Artanicus** [Artanicus/python-cozify#18]
At current cloud authentication is failing with an APIError 500:
```
%> util/auth.py
Enter your Cozify account email address: foo@bar.baz
Traceback (most recent call last):
  File "util/auth.py", line 10, in <module>  
    main()              
  File "util/auth.py", line 6, in main                                       
    assert cloud.authenticate()
  File "/home/artanicus/python-cozify.git/cozify/cloud.py", line 46, in authenticate
    cloud_api.requestlogin(email)        
  File "/home/artanicus/python-cozify.git/cozify/cloud_api.py", line 87, in requestlogin
    post('/user/requestlogin', data=payload, **kwargs)
  File "/home/artanicus/python-cozify.git/cozify/cloud_api.py", line 57, in post 
    **kwargs)           
  File "/home/artanicus/python-cozify.git/cozify/cloud_api.py", line 220, in _call
    message=response.text))                                       
cozify.Error.APIError: API error, 500: Error in server - https://cloud2.cozify.fi/ui/0.2/user/requestlogin - {"code": 1, "message": "General error"}
```

This happens regardless of email provided so is either an API change or an internal fault upstream. Escalated to upstream.

**Comment by Artanicus** [comment]
This is likely a regression resulting from the recent refactor. The API is wonky on purpose apparently, i.e. requiring a GET parameter on a POST call instead of actually POSTing the parameter. Fix incoming soon.

**Comment by Artanicus** [comment]
Fix released in cozify-0.2.28 and now live in PyPi.
